src/System.Windows.Forms/src/System/Windows/Forms/DarkModeSettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Windows.Forms
```

Comment:
For the new files it's worth considering using top level namespaces.
```suggestion
namespace System.Windows.Forms;
```